src/doc/src/reference/config.md

```diff
@@ -992,6 +992,16 @@ metadata_key2 = "value"
 
 The `[term]` table controls terminal output and interaction.
 
+##### `term.quiet`
```

Comment:
When adding a new config option, there are a couple other places in the docs that need updating:
* The TOML example at the top of this page.
* The [environment variables](https://github.com/rust-lang/cargo/blob/master/src/doc/src/reference/environment-variables.md#configuration-environment-variables) list.